src/oscar/apps/checkout/apps.py

```diff
@@ -1,24 +1,30 @@
 from django.conf import settings
 from django.conf.urls import url
 from django.contrib.auth.decorators import login_required
+from django.utils.translation import ugettext_lazy as _
 
-from oscar.core.application import Application
+from oscar.core.application import OscarConfig
 from oscar.core.loading import get_class
 
 
-class CheckoutApplication(Application):
-    name = 'checkout'
+class CheckoutConfig(OscarConfig):
+    label = 'checkout'
+    name = 'oscar.apps.checkout'
+    verbose_name = _('Checkout')
 
-    index_view = get_class('checkout.views', 'IndexView')
-    shipping_address_view = get_class('checkout.views', 'ShippingAddressView')
-    user_address_update_view = get_class('checkout.views',
-                                         'UserAddressUpdateView')
-    user_address_delete_view = get_class('checkout.views',
-                                         'UserAddressDeleteView')
-    shipping_method_view = get_class('checkout.views', 'ShippingMethodView')
-    payment_method_view = get_class('checkout.views', 'PaymentMethodView')
-    payment_details_view = get_class('checkout.views', 'PaymentDetailsView')
-    thankyou_view = get_class('checkout.views', 'ThankYouView')
+    namespace = 'checkout'
+
+    def ready(self):
+        self.index_view = get_class('checkout.views', 'IndexView')
```

Comment:
What is the reason for having to move these assignments into the `ready()` method?